src/content/docs/mlops/bring-your-own/mlops-byo.mdx

```diff
@@ -5,33 +5,66 @@ redirects:
   - /docs/alerts-applied-intelligence/mlops/bring-your-own/mlops-byo
 ---
 
-The Model Performance monitoring solution of New Relic provides a Python library that makes it easy to monitor your ML models.
-With **bring your own data (BYOD)**, you can quickly send your model's inference and metric data to the New Relic platform, directly from a notebook or any other environment.
-
-The key features:
- * Easy to integrate
- * ML framework agnostic
- * Full compatibility with New Relic observability framework
- * Based on the `newrelic-telemetry-sdk-python` library
- * Monitoring and alerting:
-    * Predictions
-    * Features
-    * Missing values
-    * Model and Data drift
-    * Custom inference data
-    * Custom performance metrics
-
-The [`ml-performance-monitoring`](https://github.com/newrelic-experimental/ml-performance-monitoring) python package, based on [`newrelic-telemetry-sdk-python`](https://github.com/newrelic/newrelic-telemetry-sdk-python), allows you to send your model's features and prediction values, as well as custom metrics, by simply adding a few lines to your code.
+New Relic's MLOps solution provides a Python library that makes it easy to monitor your machine learning models. New Relic Bring Your Own (BYO) **Machine Learning Model Monitoring** is an offering of the New Relic product suite that allows users to monitor the performance of their own machine learning models, regardless of the platform or framework they are using. This can help users identify issues with their models and take corrective action before those issues impact the accuracy or performance of their applications.
 
+In an increasingly complex digital space, data teams rely heavily on prediction engines to make decisions. New Relic Machine Learning Model Monitoring allows your team to take a step back and look at the whole picture.Our model performance monitoring allows your team to examine your ML models to identify issues efficiently and make decisions. By leveraging NR core features like setting alerts on any metrics and get notified once your ML performance has changed
+
+## Utilize our ML Framework Agnostic [#utilize-ml-framework]
```

Comment:
I am not sure what this title means

Reply:
What would you like the title to be replaced with? This was the original title in the doc you created

Reply:
This has been done.